### PR TITLE
Hamming Distance

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#350 
#### What does this implement/fix? Explain your changes.
- Hamming distance added

### CODE REVIEW

File: Document/Document.ipynb [hunk 1]
```diff
@@ -13976,13 +14025,17 @@
     "\n",
     "<blockquote id=\"ref78\">78- G. W. Brier, \"Verification of forecasts expressed in terms of probability,\" <i>Monthly weather review</i>, vol. 78, no. 1, pp. 1-3, 1950.</blockquote>\n",
     "\n",
-    "<blockquote id=\"ref79\">79- L. Buitinck et al., \"API design for machine learning software: experiences from the scikit-learn project,\" <i>arXiv preprint arXiv:1309.0238</i>, 2013.</blockquote>\n"
+    "<blockquote id=\"ref79\">79- L. Buitinck et al., \"API design for machine learning software: experiences from the scikit-learn project,\" <i>arXiv preprint arXiv:1309.0238</i>, 2013.</blockquote>\n",
+    "\n",
+    "<blockquote id=\"ref80\">80- Hamming, Richard W. \"Error detecting and error correcting codes.\" The Bell system technical journal 29.2 (1950): 147-160.</blockquote>\n",
```
Comment (sepandhaghighi) [inline Document/Document.ipynb:14030]: @alirezazolanvari Would you please check the format of the references? (80, 81)
Then I will update `README.md`

Comment (alirezazolanvari) [reply Document/Document.ipynb:14030]: Hi @sepandhaghighi. Please replace them by the followings
"<blockquote id=\"ref80\">80- R. W. Hamming, \"Error detecting and error correcting codes,\" <i>The Bell system technical journal</i>, vol. 29, no. 2, pp. 147-160, 1950.</blockquote>\n",

"<blockquote id=\"ref81\">81- S. S. Choi, S. H. Cha, and C. C. Tappert, \"A survey of binary similarity and distance measures,\" <i>Journal of systemics, cybernetics and informatics</i>, vol. 8, no. 1, pp. 43-48, 2010.</blockquote>\n"

Comment (sepandhaghighi) [reply Document/Document.ipynb]: Fix in https://github.com/sepandhaghighi/pycm/pull/443/commits/6787e41da5fde9d4582db76667709106b0e4e2f0

**Review by sadrasabouri — APPROVED** [review]
Good job! :100:

**Review by alirezazolanvari — APPROVED** [review]
LGTM!

**Pull request merged**